### ISSUE TITLE
support command: `show create sink`

### ISSUE THREAD

**Issue by neverchanje** [risingwavelabs/risingwave#9048]
### Is your feature request related to a problem? Please describe.

# RisingWave `SHOW CREATE SINK` Command

The RisingWave `SHOW CREATE SINK` command is used to display the SQL
statement that was used to create a particular sink in the current
schema. A sink is a destination for data that has been processed by a
stream processing application.

The basic syntax for the `SHOW CREATE SINK` command is as follows:

``` sql
SHOW CREATE SINK sink_name;
```

Here is a brief explanation of each element:

  - `sink_name`: This is the name of the sink for which you want to
    display the SQL statement.

If you execute the `SHOW CREATE SINK` command with a sink name,
RisingWave will display the SQL statement that was used to create that
sink. The output will include information about the sink’s name, type,
connection properties, format, and any other relevant options.

Here is an example of how to use the `SHOW CREATE SINK` command to
display the SQL statement for a sink named `metrics`:

``` sql
SHOW CREATE SINK metrics;
```

This command will display the SQL statement that was used to create the
`metrics` sink, including all of its configuration options.

In summary, the `SHOW CREATE SINK` command is a useful tool for
understanding how sinks are configured in a RisingWave application. By
using this command, you can verify the settings for a particular sink
and troubleshoot any issues that may arise.

**Comment by tabVersion** [comment]
I don't get in shock case users will use this command. Maybe we just provide more info in 'show sinks' or in 'explain create sink'
